seylii/templates/peachjam/_set_language.html

```diff
@@ -0,0 +1 @@
+<!-- djlint:off -->
```

Comment:
@nickmwangemi whats this file for?

Reply:
@actlikewill Removes the language selector